data/styles/clear/include/Basemap.mapcss

```diff
@@ -56,7 +59,8 @@ area[landuse=construction],
 area[landuse=churchyard],
 area[landuse=landfill],
 area[landuse=railway],
-area[landuse=quarry]
+area[landuse=quarry],
+area[landuse=education],
```

Comment:
More specific area like school, university, should have a priority over more generic "education".
I suggest let's move education to z-index:1 and move university, college, etc. to 2.